PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+# PULL REQUEST TEMPLATE
+
+## Description
+
+Please provide a brief description of the changes made in this pull request.
+
+## Related Issue
+
+If this pull request is related to any GitHub issue, please mention it here with the appropriate issue number.
+
+## Proposed Changes
+
+List down the specific changes or features introduced in this pull request.
+
+- Feature/issue-number/feature-description: [Feature Description]
+- Refactor/issue-number/issue-description: [Issue Description]
+- ...
```

Comment:
Burası tam açıklayıcı olmamış, pull request başlığını burada belirtelim, açıklayıcı ve ingilizce olması şartını koşalım. Açıklama kısmına opsiyonel olarak ne yaptığını anlatabilir.

Reply:
I revised this part as shown below.

## Proposed Changes

Please provide a brief and descriptive title for your pull request, summarizing the changes or features introduced. It should be in English and clearly indicate the purpose of the pull request. Optionally, you can provide a more detailed description of what you have done.

- Feature/issue-number/feature-description: [Feature Description]
- Refactor/issue-number/issue-description: [Issue Description]
- ...